package.json

```diff
@@ -10,11 +10,15 @@
     "@types/react-native": "^0.69.5",
     "eslint": "^8.21.0",
     "turbo": "^1.4.2",
-    "typescript": "^4.7.4"
+    "typescript": "^4.9.5"
   },
   "scripts": {
     "native": "cd apps/expo && expo start",
     "web": "cd apps/next && yarn next"
   },
-  "packageManager": "yarn@3.4.1"
+  "packageManager": "yarn@3.4.1",
+  "dependencies": {
+    "react-native": "^0.71.4",
+    "ts-node": "^10.9.1"
+  }
```

Comment:
If you change a dependency or a dependency version, put that in a separate `chore` commit

Reply:
Reseted the branch and created a chore commit for this